pandas/core/groupby/generic.py

```diff
@@ -1334,6 +1334,29 @@ def unique(self) -> Series:
 
 
 class DataFrameGroupBy(GroupBy[DataFrame]):
+    groupby_agg_with_tuples_example = """
+            Demonstrates using the `agg` method with a list of tuples for grouping and aggregation.
+
+            Consider a DataFrame `df`:
+
+            key | data
```

Comment:
Can you just add a code example of this in `_agg_examples_doc` below?

Reply:
Okay